rosetta-source/src/main/rosetta/event-common-func.rosetta

```diff
@@ -1433,19 +1433,25 @@ func Create_RollPrimitiveInstruction: <"Creates the primitive instructions for a
             }
         ]
 
-func Create_RollTermChangeInstruction: <"Creates the relevant terms change primitive instruction object for rolling a contractual product, which consists in the same terms as the original contractual product but with different effective and termination dates.">
+func Create_EffectiveOrTerminationDateTermChangeInstruction: <"Creates the relevant terms change primitive instruction object for rolling a contractual product, which consists in the same terms as the original contractual product but with different effective and termination dates.">
     inputs:
         contractualProduct ContractualProduct (1..1) <"The original contractual product to be rolled.">
-        effectiveRollDate AdjustableOrRelativeDate (1..1) <"The date to close and open a new position.">
-        terminationDate AdjustableOrRelativeDate (1..1) <"The new termination date.">
+        effectiveRollDate AdjustableOrRelativeDate (0..1) <"The date to close and open a new position.">
+        terminationDate AdjustableOrRelativeDate (0..1) <"The new termination date.">
```

Comment:
Given that both arguments are now optional, consider adding a condition that checks that at least one of them exists.

Reply:
Fixed. Added condition.